src/windowsservercore/ltsc2022/helix/webassembly-net8/amd64/Dockerfile

```diff
@@ -38,7 +38,7 @@ RUN cd %EMSDK_PATH% \
     && .\upstream\emscripten\embuilder.bat build MINIMAL
 
 # install Node JS
-ENV NODE_VERSION 17.3.1
+ENV NODE_VERSION 18.20.5
```

Comment:
Why is this only being bumped to version 18 when there's a version 22 LTS available for much longer support?

Reply:
I wanted to limit the risk for now.